active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
Without a test for progress_bar true, you are adding a run-time dependency that may or may not work on future updates. Personally, I don't believe the progress bar is worth adding, given that you could mimic this behavior with `STDOUT.puts` or `logger.info`.